src/xcode/ENA/ENA/Source/Services/DSCList/DSCListProvider.swift

```diff
@@ -0,0 +1,87 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import OpenCombine
+import HealthCertificateToolkit
+
+final class DSCListProvider: DSCListProviding {
+
+	// MARK: - Init
+
+	init(
+		client: DSCListFetching,
+		store: Store,
+		interval: TimeInterval = 12 * 60 * 60  // 12 hours
```

Comment:
Thanks for defining it this way, gets so much easier to unterstand the magic number :)